hedera-node/hedera-mono-service/src/main/java/com/hedera/services/ledger/accounts/AliasManager.java

```diff
@@ -240,7 +244,30 @@ public void forgetEvmAddress(final ByteString alias) {
      * @return EntityNum mapped to the given alias.
      */
     public EntityNum lookupIdBy(final ByteString alias) {
-        return curAliases().getOrDefault(alias, MISSING_NUM);
+        var entityNum = curAliases().getOrDefault(alias, MISSING_NUM);
+        if (alias.size() > EVM_ADDRESS_LEN && entityNum == MISSING_NUM) {
+            // if we don't find entity num for key alias we can try to derive EVM address from it
+            // and look it up
+            try {
+                final Key key = Key.parseFrom(alias);
+                final JKey jKey = JKey.mapKey(key);
+                final var evmAddress = tryAddressRecovery(jKey, ADDRESS_RECOVERY_FN);
+                if (evmAddress != null) {
+                    entityNum =
+                            curAliases().getOrDefault(ByteString.copyFrom(evmAddress), MISSING_NUM);
+                }
+            } catch (InvalidProtocolBufferException
```

Comment:
Should be easy cover at least the `InvalidProtocolBufferException` branch of this multi-catch.